### PR TITLE
🕴️ Move to API v0.3

### LINKED ISSUE #132: Move to API v0.3
## Learning objectives

- Objective

## Todo

These points are a rough guideline. **Please** feel free to discuss with others on the team about the best way to design this component!

- Todo

## Useful links

- Useful link

### PR DESCRIPTION
**Pull request by Qrtn**
## Status:
:rocket: Ready

## Description

- Moves to API v0.3
- Adds loading skeletons in mobile list view
- Passes true total count so that you can scroll longer
- Self-hosts Lato font
- Consolidate Google Maps loaders between use-places-autocomplete and google-map-react (use js-api-loader)

Fixes #132

## Todos

- Add support for total counts from API

## Screenshots

image

### CODE REVIEW

File: src/components/entry/EntryOverview.js [hunk 1]
```diff
@@ -120,7 +120,8 @@ const EntryOverview = ({ locationData, className }) => {
     </TagList>
   )
 
-  const allTypeNames = locationData && locationData.type_names.join(', ')
+  const allTypeNames =
+    locationData && locationData.type_ids.map(getTypeName).join(', ')
```
Comment (SirajChokshi) [inline src/components/entry/EntryOverview.js:124]: could be more concise with optional chaining

File: src/components/list/EntryList.js [hunk 2]
```diff
@@ -23,24 +23,18 @@ const StyledListEntry = styled(ListEntry)`
...
-      ...props
-    },
+    { locations, onEntryClick, onEntryMouseEnter, onEntryMouseLeave, ...props },
     ref,
   ) => {
     const renderRow = ({ index, style }) => {
       const location = locations[index]
-      let row = null
+
+      let row
```
Comment (SirajChokshi) [inline src/components/list/EntryList.js:37]: any reason we're no longer using null here?

File: src/utils/api.ts [hunk 3]
```diff
@@ -21,99 +21,102 @@ const handleResponse = (request: Promise<AxiosResponse<any>>) =>
...
     const formData = new FormData()
     return formData.append('photo_data', photoData)
   }
   return null
 }
+*/
 
 export const getClusters = (
-  params: paths['/clusters.json']['get']['parameters']['query'],
+  params: paths['/clusters']['get']['parameters']['query'],
```
Comment (SirajChokshi) [inline src/utils/api.ts:35]: should we remove these last 3 property accessors in favor a `.` operator?

**Review by SirajChokshi — APPROVED** [review]
🚀  Great work!! Left a few comments

File: src/components/list/EntryList.js (hunk 2 above)
Comment (Qrtn) [reply src/components/list/EntryList.js]: Yes, it's an if/else clause now, so always will be filled in.

File: src/components/entry/EntryOverview.js (hunk 1 above)
Comment (Qrtn) [reply src/components/entry/EntryOverview.js]: Clever girl!

File: src/utils/api.ts (hunk 3 above)
Comment (Qrtn) [reply src/utils/api.ts:35]: I believe you can't use dot notation here since paths is a TS interface.

**Pull request merged**